README.md

```diff
@@ -82,3 +82,102 @@ or the Gitter channel:
 https://gitter.im/HenrikJoreteg/SimpleWebRTC
 
 ## API
+
+### Constructor
+
+`new SimpleWebRTC(options)`
+
+- `object options` - options object provided to constructor consisting of:
+  - `string url` - *required* url for signaling server. Defaults to signaling server URL which can be used for development. You must use your own signaling server for production.
+  - `object sockio` - *optional* object to be passed as options to the signaling server connection.
```

Comment:
isn't it socketio?  see: https://github.com/andyet/SimpleWebRTC/blob/master/socketioconnection.js#L4